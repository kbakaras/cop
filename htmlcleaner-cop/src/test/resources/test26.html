<html xmlns="http://www.w3.org/TR/REC-html40">
<body>

<table>

 <tr>
  <td>Col1</td>
  <td>Col2</td>
 </tr>
 <tr>
  <td>Content</td>
  <td>Content</td>
 </tr>

</table>
</body></html>