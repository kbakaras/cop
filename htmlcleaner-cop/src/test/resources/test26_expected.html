<html>
<head />
<body>


<table>

 <tbody><tr>
  <td>Col1</td>
  <td>Col2</td>
 </tr>
 <tr>
  <td>Content</td>
  <td>Content</td>
 </tr>

</tbody></table>
</body></html>